### ISSUE TITLE
[Bug]: some pages have skipped header levels

### ISSUE THREAD

**Issue by aaaaargZombies** [geeksforsocialchange/gfsc-v3#341]
## Description

sorry this is a quick note to remind us to investigate and document more clearly

## Steps to reproduce

1. What you did first
2. What you did next

## Platform (if relevant)

What device and browser were you using?

**Comment by ivan-kocienski-gfsc** [comment]
This seems related to #340 a bit.

**Comment by kimadactyl** [comment]
was curious the impact of this and it seems like it might be a myth?

https://accessible-digital-documents.com/blog/the-skipped-headings-myth/

> One of the most persistent myths amongst document producers is that skipping a heading level will always create an accessibility problem. Whilst many well-structured documents do follow a strictly descending hierarchical heading structure (ie without skipping a level), sometimes skipping a heading level is the only sensible thing to do.
> A good starting point in debunking this myth is WCAG (the Web Content Accessibility Guidelines). Quite simply, there is no prohibition on skipped headings in WCAG. It’s just not there, and never has been. The WCAG position was neatly summed up by Bruce Bailey, a long-serving invited expert on the [W3C WAI GL](http://www.w3.org/WAI/GL/) working group, who explained that in the course of drafting WCAG 2.0:

**Comment by katjam** [comment]
It is useful to consider skipped headings when they are flagged to make sure the semantics are correct. It's not a failure of Accessibility standards to skip a heading but they should be meainingful in realtion to each other. I think a lot of the guidance around "skipped headings = bad" stems from people using the tags as style helpers. I still come across this a lot where a designer specifies set styles for h1-h6 as a range of ways to visually display headings and then they get applied for pure visual rather than hierarchy which is wrong / inaccessible and often manifests as skipped headings.

**Comment by kimadactyl** [comment]
Yeah I get why it happens and think it's prob one of the main issues with using Markdown to maintain a blog in.

I'm thinking through this practically though and I don't think we violate the skipped rule. Generally it should be that page titles are h1s. Then if it's a shorter article with just a couple of subheadings I think the current h3s look better as h2 can look way too big and super awkward design wise. h2 feels more like a chapter heading so it tends to just go in longer pieces with hrs or images breaking it up too.

In other words - it either goes h1 > h2 > h3 for long pieces or h1 > h3 for short ones, we never go h1 > h3 > h2. So I think we are covered but just one to keep an eye out for and maybe make sure is in the style guide? (of course this could be wrong but from memory this is why the headings may be used differently in diff articles)

**Comment by aaaaargZombies** [comment]
That's interesting about the PDFs hopefully that's true for websites too

> we never go h1 > h3 > h2.

I think my main concern was noticing things starting to drift out of sync and thinking its a bit of a foot gun that will make the above easier.

Maybe we need a couple of differently styled header shortcodes if the design/heading level are causing conflicts of interest?
